README.md

```diff
@@ -323,6 +324,20 @@ Creates an array of unique values that are included in all given arrays.
 
 **[⬆️ back to top](#available-helpers)**
 
+#### `invoke`
```

Comment:
Is it just me or are these additions strangely highlighted?

<img width="1006" alt="screenshot 2016-03-22 11 03 53" src="https://cloud.githubusercontent.com/assets/1390709/13940988/d5eb459a-f01d-11e5-8bd3-40c2a65c6393.png">